### PR TITLE
[Inference] integrate  deepseek-coder-33b-instruct.

### PR DESCRIPTION
**Pull request by yao531441**

### CODE REVIEW

File: llm_on_ray/inference/models/deepseek-coder-33b-instruct.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+port: 8000
+name: deepseek-coder-33b-instruct
+route_prefix: /deepseek-coder-33b-instruct
+num_replicas: 1
+cpus_per_worker: 24
+gpus_per_worker: 0
+deepspeed: false
+workers_per_group: 2
+device: "cpu"
```
Comment (carsonwang) [inline llm_on_ray/inference/models/deepseek-coder-33b-instruct.yaml:9]:
```suggestion
device: cpu
```

**Review by carsonwang — COMMENTED** [review]
Does this 33b model run well on CPU?

File: llm_on_ray/inference/models/deepseek-coder-33b-instruct.yaml (hunk 1 above)
Comment (yao531441) [reply llm_on_ray/inference/models/deepseek-coder-33b-instruct.yaml]: The latency of this model significantly exceeds that of the 7b model; however, it outperforms other models, including Codellama, in the specific task of converting Scala code to Velox UDF.

File: llm_on_ray/inference/models/deepseek-coder-33b-instruct.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+port: 8000
+name: deepseek-coder-33b-instruct
+route_prefix: /deepseek-coder-33b-instruct
+num_replicas: 1
+cpus_per_worker: 24
+gpus_per_worker: 0
+deepspeed: false
+workers_per_group: 2
+device: CPU
```
Comment (KepingYan) [inline llm_on_ray/inference/models/deepseek-coder-33b-instruct.yaml:9]: Please change this to lowercase letters.

**Comment by carsonwang** [comment]
@yao531441 , thanks! can you please merge main branch and we will merge this.

**Comment by yao531441** [comment]
> @yao531441 , thanks! can you please merge main branch and we will merge this.

Have rebased main.

**Review by carsonwang — APPROVED** [review]

**Pull request merged**